### PR TITLE
Test cleanups

### PR DESCRIPTION
**Pull request by phlax**
These tests attempt to write to the testing systems pootle.conf

### CODE REVIEW

File: tests/commands/pootle_runner.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ def test_pootle_version(capfd):
 
 
 @pytest.mark.cmd
-def test_pootle_init(capfd):
+@pytest.mark.django_db
+def __test_pootle_init(capfd):
     """pootle init --help"""
     call(['pootle', 'init', '--help'])
```
Comment (dwaynebailey) [inline tests/commands/pootle_runner.py]: This doesn't write out a ``pootle.conf`` file. So not 100% sure why its needs disabling

File: tests/commands/pootle_runner.py [hunk 2]
```diff
@@ -38,7 +39,7 @@ def test_pootle_init(capfd):
 
 
 @pytest.mark.cmd
-def test_pootle_init_db_sqlite(capfd):
+def __test_pootle_init_db_sqlite(capfd):
     """pootle init --help"""
     call(['pootle', 'init', '--db=sqlite'])
```
Comment (dwaynebailey) [inline tests/commands/pootle_runner.py]: We can write a temporary files using ``--config``

File: tests/commands/pootle_runner.py (hunk 1 above)
Comment (phlax) [inline tests/commands/pootle_runner.py]: apologies - i probs disabled it in error

File: tests/commands/pootle_runner.py (hunk 2 above)
Comment (dwaynebailey) [inline tests/commands/pootle_runner.py]:
```diff
diff --git a/tests/commands/pootle_runner.py b/tests/commands/pootle_runner.py
index 2dba648..f4258f8 100644
--- a/tests/commands/pootle_runner.py
+++ b/tests/commands/pootle_runner.py
@@ -38,8 +38,9 @@ def test_pootle_init(capfd):
 
 
 @pytest.mark.cmd
-def test_pootle_init_db_sqlite(capfd):
+def test_pootle_init_db_sqlite(capfd, tmpdir):
     """pootle init --help"""
-    call(['pootle', 'init', '--db=sqlite'])
+    test_conf_file = tmpdir.join("pootle.conf")
+    call(['pootle', 'init', '--db=sqlite', '--config=%s' % test_conf_file])
     out, err = capfd.readouterr()
     assert "Configuration file created" in out
```

**Comment by dwaynebailey** [comment]
I'm no expert, but lgtm

**Comment by phlax** [comment]
> Mmm I really prefer if email is not None

it really doesnt matter in this context. testing for is/not ``None`` is essential in a certain type of situation - ie where the var has the potential to have a zero value other than none - and you do/not want to use the var in that situation.

**Pull request merged**